ide/projectapi/manifest.mf

```diff
@@ -1,6 +1,6 @@
 Manifest-Version: 1.0
 OpenIDE-Module: org.netbeans.modules.projectapi/1
-OpenIDE-Module-Specification-Version: 1.77
+OpenIDE-Module-Specification-Version: 1.78
 OpenIDE-Module-Localizing-Bundle: org/netbeans/modules/projectapi/Bundle.properties
 OpenIDE-Module-Layer: org/netbeans/modules/projectapi/layer.xml
-OpenIDE-Module-Recommends: cnb.org.netbeans.modules.projectapi.nb
+OpenIDE-Module-Needs: org.netbeans.spi.project.ProjectManagerImplementation
```

Comment:
@tzezula: Here we need an implementation of `org.netbeans.spi.project.ProjectManagerImplementation`. Previously the module recommended single implementation, but that wasn't enough. If the module was missing a `NullPointerException` was raised. Hence we "need" - e.g. the module isn't going to be enabled without an implementation, but the implementation can come from any module.